### PR TITLE
Fix ES and Kibana Bring up

### PR DESCRIPTION
**Pull request by ashwinikmp**

### CODE REVIEW

File: elasticsearch/values-beta.yaml [hunk 1]
```diff
@@ -22,24 +22,22 @@ roles:
   - transform
 
 replicas: 3
-minimumMasterNodes: 3
+minimumMasterNodes: 2
```
Comment (een-anoopkp) [inline elasticsearch/values-beta.yaml:25]: why is this count reduced to 2. This can lead to split-brain condition, right?

File: elasticsearch/values-beta.yaml [hunk 2]
```diff
@@ -48,28 +46,54 @@ esJvmOptions:
...
+#    path: /usr/share/elasticsearch/config/certs
+#    defaultMode: 0755
+
+hostAliases: []
+#- ip: "127.0.0.1"
+#  hostnames:
+#  - "foo.local"
+#  - "bar.local"
+
+image: "docker.elastic.co/elasticsearch/elasticsearch"
```
Comment (een-anoopkp) [inline elasticsearch/values-beta.yaml:66]: Why are these lines added back again?

File: elasticsearch/values-beta.yaml [hunk 3]
```diff
@@ -122,52 +137,23 @@ extraInitContainers:
...
+  publishNotReadyAddresses: false
   nodePort: "30013"
 
 # This is the max unavailable setting for the pod disruption budget
 # The default value of 1 will make sure that kubernetes won't allow more than 1
 # of your pods to be unavailable during maintenance
 maxUnavailable: 1
-
-nodeSelector: 
-   node-class: es-beta
```
Comment (een-anoopkp) [inline elasticsearch/values-beta.yaml:173]: Why is the nodeselector removed and being passed in the command as argument?

File: elasticsearch/values-beta.yaml (hunk 1 above)
Comment (ashwinikmp) [reply elasticsearch/values-beta.yaml:25]: No. We are now allowing atleast 1 master node to go down. Then rest 2 nodes should decide who the master is.

File: elasticsearch/values-beta.yaml (hunk 2 above)
Comment (ashwinikmp) [reply elasticsearch/values-beta.yaml:66]: Thanks. I will remove it in next PR

File: elasticsearch/values-beta.yaml (hunk 3 above)
Comment (ashwinikmp) [reply elasticsearch/values-beta.yaml:173]: I will put it back

Comment (ashwinikmp) [reply elasticsearch/values-beta.yaml]: done

**Review by shubham-een — COMMENTED** [review]
looks like all the SSL parts have been removed. 
is not required in 8.1.1 or this disables the security ?

**Review by shubham-een — APPROVED** [review]

**Review by een-anoopkp — APPROVED** [review]

**Comment by ashwinikmp** [comment]
Redployed after the review changes done. Going ahead with merge
